packages/edit-site/src/components/sidebar/global-styles-v2/components/style-preview.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { HStack, Spacer, View, VStack } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useAppState } from '../state';
+
+export const StylePreview = () => {
+	const { get } = useAppState();
+	const fontFamily = get( 'typography.fontFamily' );
```

Comment:
When figuring out the state architecture, we should ensure to make these more developer-friendly. This looks error-prone.

Reply:
With TypeScript it should be theoretically possible to prevent errors against that `get` method but I prefer to just use regular selectors over these string selectors (though they're already used for editor settings, right?)